### DISCUSSION TITLE
System-nsm | no space left on device

### DISCUSSION THREAD

**Discussion by RoyTM99** [Security-Onion-Solutions/securityonion#4910] [Unsupported Versions] [answered]
Hi Everyone,

We have been working fine with Secuirty Onion 2.3.30 until now, however since a few days ago we noticed that we stopped receiving alerts, when running a so-status we observed an error in so-steno, investigating we found that we have 100% of our /dev/mapper/system-nsm, what can we do to purge it?

we have run the commands so-elastic-clear and so-sensor-clean, this has reset the so-steno service to ok and eliminated a couple of GB, however in the system-nsm it is still at 100%.

Validating with du we found that the elasticsearch indexes are the ones that fill most of the space, is it possible to delete the folders of these indexes or we could have some kind of affectation? is there any other way to purge those files?

Elast Version: 7.10.2
Security Onion: 2.3.30

Thank you
Kind Regards

**Comment by thegreekman** [comment]
I  have the same issue and question

**Reply by dougburks** [reply]
@thegreekman Please see @gcsrellis response below. If you have further questions or problems, please start a separate discussion as described at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720. Thanks!

**Comment by gcsrellis** [comment, marked as answer]
> Hi Everyone,
> 
> We have been working fine with Secuirty Onion 2.3.30 until now, however since a few days ago we noticed that we stopped receiving alerts, when running a so-status we observed an error in so-steno, investigating we found that we have 100% of our /dev/mapper/system-nsm, what can we do to purge it?
> 
> we have run the commands so-elastic-clear and so-sensor-clean, this has reset the so-steno service to ok and eliminated a couple of GB, however in the system-nsm it is still at 100%.
> 
> Validating with du we found that the elasticsearch indexes are the ones that fill most of the space, is it possible to delete the folders of these indexes or we could have some kind of affectation? is there any other way to purge those files?
> 
> Elast Version: 7.10.2
> Security Onion: 2.3.30
> 
> Thank you
> Kind Regards

I believe the issue was fixed in a recent version.  Updating to the latest should resolve it.  Read the patch notes for the releases since 2.3.30 and it should have the information in there.
